src/Camera.cpp

```diff
@@ -1,29 +1,57 @@
-
 #include "Camera.h"
 
 #include <GLFW/glfw3.h>
 #include <glm/gtc/matrix_transform.hpp>
 #include <glm/mat4x4.hpp>
+#include <glm/glm.hpp>
+#define GLM_ENABLE_EXPERIMENTAL 1
+#include <glm/gtx/transform.hpp>
 
 
-
-Camera::Camera()
-	: ProjectionMatrix(glm::mat4(1.0))
-	, ViewMatrix(glm::mat4(1.0))
+Camera::Camera(glm::vec3 p, glm::vec3 f, glm::vec3 u):
+	pos(p) 
+, forward(f)
+, up(u)
 {
-	// Initial position
-	glm::vec3 position = glm::vec3(4,3,3);
-	// Initial Field of View
-	float initialFoV = 45.0f;
 
-	ViewMatrix = glm::lookAt(
-		position, // Camera position in World Space
-		glm::vec3(0,0,0), // and looks at the origin
-		glm::vec3(0,1,0)  // Head is up (set to 0,-1,0 to look upside-down)
+	ViewMatrix = glm::lookAt
+	(
+		pos, // Camera position in World Space
+		forward, // and looks at the origin
+		up  // Head is up (set to 0,-1,0 to look upside-down)
 	);
 
 	//perspective (T const &fovy, T const &aspect, T const &near, T const &far)
-	ProjectionMatrix = glm::perspective(glm::radians(initialFoV), 4.0f / 3.0f, 0.1f, 100.0f);
+	float initialFoV = 45.0f;
+	float near = 0.1f;
+	float far = 100.f;
+	float aspect = 1.0;
```

Comment:
[Minor issue] Can make these const variables as well. Prob better to keep these as Camera members in header.